campari/RomanASP.py

```diff
@@ -217,6 +222,39 @@ def main():
     assert grid_type in ["regular", "adaptive", "contour",
                          "single", "none"], er
 
+    # Option 1, user passes a file of SNIDs
+    if args.SNID_file is not None:
+        SNID = pd.read_csv(args.SNID_file, header=None).values.flatten().tolist()
+
+    # Option 2, user passes a SNID
+    elif args.SNID is not None:
+        SNID = args.SNID
```

Comment:
This logic is inconsistent with the documentation.  The above lines of code mean that `--SNID` overrides `--SNID_file`, but the documentation says the opposite:

```
    parser.add_argument("-s", "--SNID", type=int, default=None,
                        required=False, nargs="*",
                        help="OpenUniverse2024 Supernova IDs; ignored if"
                             " --SNID-file is given")
    parser.add_argument("--SNID-file", type=str, default=None, required=False,
                        help="Path to a csv file containing a list of "
                             "OpenUniverse SNIDs to run.")
```